src/components/cube.jsx

```diff
@@ -28,6 +32,41 @@ const Cube = forwardRef(({ app, tableData: { initialColumns }, closeOnClickOutsi
   let model = null;
   let hypercubeProps = null;
 
+  const modifyLocalStorage = (action) => {
```

Comment:
Perhaps use something like https://github.com/rehooks/local-storage ?

Reply:
I don't want to constantly read/write to localstorage (to keep sync with some kind of state). Otherwise it's just replacing one line with another line and also adding a new dependency.